exasol/analytics/query_handler/graph/stage/sql/execution/input.py

```diff
@@ -10,6 +10,12 @@
 
 @dataclasses.dataclass(frozen=True, eq=True)
 class SQLStageGraphExecutionInput:
+    """
+    The class is an input for the graph query handler (not to be confused with
+    a node query handler provided by a user).
```

Comment:
I am not familiar with the term "node query handler". 
Additionally, I propose, to add comments about the graph query handler to its class definition rather than to the `SQLStageGraphExecutionInput`.

Reply:
There are two query handlers - one for the graph, implemented here, and one (or many) for each node, provided by the user.

Reply:
There a docstring for the `SQLStageGraphExecutionQueryHandler`, albeit a short one.